README.md

```diff
@@ -29,7 +29,7 @@ Over and and above the tasks specified below, there are particular aspects of ea
 * Data Engineer candidates - as the key enablers of our unit's work, we really want to see work done in a sustainable manner: writing for easy comprehension, testing, clean code, modularity all bring us joy.
 * Data Analyst candidates - we consider success for our analysts when they provide the insights that inform actual decisions. Hence, we want evidence of both the ability to surface these insights from data, as well as the rhetorical skill in conveying the implications thereof. Your audience is intelligent, but non-specialist.
 
-### Candidates where programming is required (Data Scientist and Engineers)
+### Candidates where programming is required (Data Scientist; Engineers & Visualisation Engineer)
 Requirements and notes:
 * Our primary programming languages are `python` and `R`. We will accept code that is packaged in `.py`, `.ipynb`, `.R` and `.Rmd` files. 
```

Comment:
Does this not need to changed to include technologies that visualisation engineer applicant could use to build there visualisation dashboards? Or add a comment for vis engineer to choose an open source technology of their liking. It reads a bit limited now with only `python` and `R`

Reply:
```suggestion
* For Data Science and Data Engineering our primary programming languages are `python`, `R` and `SQL`. We will accept code that is packaged in `.py`, `.ipynb`, `.R` and `.Rmd` files. Scripts in `.sql` may also be included where applicable.
* Data Visualisation engineers should have knowledge `python` or `R`, and a relevant front-end programming language (Javascript and equivalents, HTML and CSS). We will accept code that is packaged in `.py`, `.R` and appropriate front-end programming language specific files, including `.js`, `.html` etc.
```